src/components/RichTextEditor/RichTextEditor.stories.tsx

```diff
@@ -135,6 +136,24 @@ const value = [
             },
         ],
     },
+    {
```

Comment:
I'd move the example values that are used in the stories into a separate file to keep the story file short and organised, wdyt?

Reply:
Good input, refactored it :)